openff/toolkit/_tests/test_examples.py

```diff
@@ -55,7 +55,7 @@ def find_example_scripts():
 
     Returns
     -------
-    example_file_paths : List[str]
+    example_file_paths : list[str]
```

Comment:
I think we're better off putting these in the actual function signature, and not using type annotations in docstring `Returns` or `Parameters` sections at all - Sphinx will render it the same way in either case. But that can come in a later PR. Or is there some reason I don't know about to do things this way in tests?

Reply:
I don't think you're missing anything - I change lines like these out of continuity, not out of some holiness of these docstring annotations. Looking back into my crystal ball, I think I picked up this habit (and/or the PyData ecosystem uses it) for historical reasons dating back long before Python had typing.

I'd be thrilled to submit a PR that removes _these_ annotations, or at least most of them where they can go in the function signature. I'm less excited about doing that here, preferring either leaving these changes in or reverting them.

Reply:
Cool let's leave that for a future PR then!